code/game/objects/items/tanks/watertank.dm

```diff
@@ -287,7 +287,7 @@
 			return
 	return
 
-/obj/item/extinguisher/mini/nozzle/interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
+/obj/item/extinguisher/mini/nozzle/ranged_interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
 	if(AttemptRefill(interacting_with, user))
 		return NONE
```

Comment:
If I'm not mistaken you're allowing these to refill at infinite range now so that has to be addressed

```dm
/nozzle/interact()
	if(refill())
		return NONE
	return ranged_interact()

/nozzle/ranged_interact()
	// the existing code
```

Reply:
Oh double checking the code refill has an inbuilt adjacency check (that's stupid) we should still do this though